ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/backtotop/backtotop.html

```diff
@@ -0,0 +1,5 @@
+<div class ='backtotop-cmp'>
```

Comment:
@ReenaOmana : Please add sample text/image for edit mode when component is added

Reply:
Have added the text in edit mode as per the comment